public/js/core/geolocation.js

```diff
@@ -12,6 +12,7 @@ export default class Geolocation {
     this._hasHeadingMarker = false;
     this._lastAutomaticZoom = null;
     this._map = map;
+    this._onOrientationFound = this._onOrientationFound.bind(this);
```

Comment:
I'd prefer to make a `.bind(this)` at both 113 and 129 lines. Locking a context by reassigning a variable to itself always felt weird to me.

Reply:
Understood that this is a little odd, however, if we bind on both lines 113 and 129 the event listener will not be removed, since I believe every call to bind returns a new function.

Reply:
Oh yes you're right, silly me...